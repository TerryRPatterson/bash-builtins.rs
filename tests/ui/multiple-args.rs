use bash_builtins::BuiltinOptions;

#[derive(BuiltinOptions)]
enum Opt {
    #[opt = 'a']
    A,

    #[opt = 'b']
    B(String),

    #[opt = 'c']
    C(String, String),
}

fn main() {}
