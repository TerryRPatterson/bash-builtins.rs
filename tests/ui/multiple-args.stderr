error: Options must have only one argument
  --> $DIR/multiple-args.rs:11:5
   |
11 |     #[opt = 'c']
   |     ^
